code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -833,6 +833,47 @@
 	new /obj/item/weapon/bonegel(location)
 	qdel(holder.my_atom)
 
+/datum/chemical_reaction/bonemilk
+	name = "Bone fixer"
+	id = BONEFIXER
+	result = BONEFIXER
+	required_reagents = list(MILK = 10, CARPOTOXIN = 5, CLONEXADONE = 10, PHOSPHORUS = 10)
```

Comment:
If this was some xenobotany or xenoarch exclusive chem maybe but really?
You can make it without leaving medbay?

Reply:
Well, maybe if medbay didn't start with the 3 gating chems that have been feature creeped in, we wouldn't be in this mess